docs/architecture/decisions/0002-dists-can-never-change-retroactively.md

```diff
@@ -0,0 +1,23 @@
+# 2. Distributions Can Never Change Retroactively
+
+**Date:** 2023-09-22
+
+## Status
+
+Accepted (for the GitHub package pipeline)
+
+## Context
+
+The Garden Linux apt repository produces daily distributions. 
+These distributions are assembled through a dedicated package repository pipeline. 
+Garden Linux maintainers define the package specifications for upcoming apt distributions.
```

Comment:
> upcoming apt distributions.

What does 'distributions' mean in this case? Is it the same as above?

This might be debian lingo that I'm not too familiar with. In debian, a distro is something like 'bookworm' or 'trixie', right?